### ISSUE TITLE
ter-padded-blocks and JSDoc

### ISSUE THREAD

**Issue by sanex3339** [buzinas/tslint-eslint-rules#296]
With new `ter-padded-blocks` option with `never` value i getting false-positive alerts on empty blank lines when JSDoc block is existing above method of the class.

![image](https://user-images.githubusercontent.com/9210534/36639750-58289468-1a24-11e8-855d-1d1154e7c56c.png)

**Comment by buzinas** [comment]
Hey @sanex3339, thanks for reporting this issue!

The fix for this one shouldn't be too hard to implement, would you like to create a PR for that?

**Comment by ryhumble** [comment]
I'll add that this occurs for all comment lines. For example, the following examples fail:
```
import * as React from 'react';

// Some comment on my class
export class MyClass extends React.Component {
  ...
}
```

```
import * as React from 'react';

export class FirstClass extends React.Component {
  ...
}

// tslint:disable-next-line max-classes-per-file
export class MyClass extends React.Component {
  ...
}
```

**Comment by tlaskowski** [comment]
Hi @buzinas , what is the status of JSDoc issue?

**Comment by micky2be** [comment]
Still having an issues with a line of comment before a block

**Comment by jmlopez-rod** [comment]
Same here:
```
@Annotation()
class A implements B<C> {
  method(a: IInterfaceA): IInterfaceB {
  }
}
```

**Comment by jmlopez-rod** [comment]
Ok, just letting you guys know that I'm working on it. There are several issues with it.

**Comment by tlaskowski** [comment]
Hi, when can we expect a new version with padded blocks fixer ?

**Comment by jmlopez-rod** [comment]
Sorry, just released version 5.4.0 last night.
